src/styles/layouts/comm-layout.scss

```diff
@@ -1,58 +1,117 @@
 .comm-hero {
   margin-top: $base-height * 8;
-}
 
-.comm-header {
-  margin-bottom: $base-height * 4;
-  font-size: 28px;
-  line-height: $base-height * 5;
-  color: $white;
-  text-align: center;
+  .comm-header {
+    margin-bottom: $base-height * 4;
+    font-size: 28px;
+    line-height: $base-height * 5;
+    color: $white;
+    text-align: center;
 
-  @include media-breakpoint-up(lg) {
-    text-align: left;
+    @include media-breakpoint-up(lg) {
+      text-align: left;
+    }
   }
-}
 
-.comm-container {
-  padding: 0;
-  text-align: center;
+  .comm-container {
+    padding: 0;
+    text-align: center;
 
-  @include media-breakpoint-up(lg) {
-    text-align: left;
+    @include media-breakpoint-up(lg) {
+      text-align: left;
+    }
   }
-}
 
-.comm-text {
-  margin-bottom: $base-height * 6;
-  color: $white;
-  text-align: center;
+  .comm-text {
+    margin-bottom: $base-height * 6;
+    color: $white;
+    text-align: center;
 
-  @include media-breakpoint-up(lg) {
-    text-align: left;
+    @include media-breakpoint-up(lg) {
+      text-align: left;
+    }
   }
+
+  .btn-forum {
+    padding: 0 $base-height * 7;
+    background-color: $aurora-blue;
+    color: $white;
+    font-family: 'Poppins';
```

Comment:
I see we have variable for font-family. Maybe we can use those in here

Reply:
Yup, update made.